### PR TITLE
Task/jenkins config

### PR DESCRIPTION
**Pull request by iguannago**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+sonar.projectName=RPE :: expressjs template
+sonar.projectKey=RPEET
+sonar.language=typescript
+sonar.sources=src/main
+sonar.tests=src/test
+sonar.test.exclusions=test/coverage/**
+sonar.typescript.lcov.reportPaths=test/coverage/html/lcov.info
+sonar.exclusions=src/main/app.ts,src/main/development.ts,src/main/index.js,src/main/modules/**,src/main/router/routerFinder.ts,src/main/server.ts
```
Comment (timja) [inline sonar-project.properties:8]: modules and router finder shouldn't really be excluded.

File: sonar-project.properties [hunk 2]
```diff
@@ -0,0 +1,8 @@
+sonar.projectName=RPE :: expressjs template
```
Comment (timja) [inline sonar-project.properties:1]: are these replaced by the init script?

File: sonar-project.properties [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+sonar.projectKey=RPEET
```
Comment (timja) [inline sonar-project.properties:2]: seems like a weird key?

File: package.json [hunk 4]
```diff
@@ -14,7 +14,7 @@
     "test": "jest",
     "test:coverage": "jest --coverage",
     "test:routes": "jest -c jest.routes.config.js",
-    "test:a11y": "jest -c jest.a11y.config.js --maxWorkers 15",
+    "test:a11y": "yarn build && jest -c jest.a11y.config.js --maxWorkers 15",
```
Comment (timja) [inline package.json:17]: this should probably be done either as a postinstall hook or in the jenkinsfile in an after build callback i think

Comment (iguannago) [reply package.json:17]: can we leave it as it is just for now and improve it later on when we get some more time?

File: sonar-project.properties (hunk 2 above)
Comment (iguannago) [reply sonar-project.properties:1]: Not, it is not. Hopefully, it's something we can improve later on as well.

File: sonar-project.properties (hunk 3 above)
Comment (iguannago) [reply sonar-project.properties:2]: I follow the same naming convention that in I&A.

Comment (timja) [reply sonar-project.properties:2]: I think this would be better?
```suggestion
sonar.projectKey=rpe-expressjs-template
```

**Pull request merged**